### PR TITLE
Unfolding updates

### PR DESCRIPTION
**Pull request by davidwalter2**
- Fix bug in mw histmaker where columns and axes previously used incorrectly with or without gen axes
- Adding a separate workflow for unfolding to clean up the CI script and allow more easily dedicated settings. This includes scheduled builds of the full unfolding (on different days than the main CI).
- This PR is on top of #292

### CODE REVIEW

**Comment by davidwalter2** [comment]
Ready for review/merge

File: .github/workflows/unfolding.yml [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+    - cron: '0 3 * * 1,3,5' # Run on Monday, Wednesday, and Friday morning at 3h00 UTC
+    - cron: '30 5 * * 1-5' # Run on weekdays
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env: 
+  MAX_FILES: 10
+  LUMI_SCALE: 760
+  NTHREADS: 16
+  LOCAL_WEB_DIR: www/WMassAnalysis/PRValidation
```
Comment (kdlong) [inline .github/workflows/unfolding.yml:20]: Is there any way to store common variables like this in a common file that both workflows read from?

File: wremnants/CardTool.py [hunk 2]
```diff
@@ -923,7 +923,7 @@ def writeHist(self, h, proc, syst, setZeroStatUnc=False, decorrByBin={}, hnomi=N
             return
         if self.project:
             axes = self.project[:]
-            if "charge" in h.axes.name and not self.xnorm:
+            if "charge" in h.axes.name and not self.xnorm and "charge" not in axes:
```
Comment (kdlong) [inline wremnants/CardTool.py:926]: I think there is a self.charge_ax (or there should be). We should avoid hardcoding it

File: .github/workflows/unfolding.yml (hunk 1 above)
Comment (davidwalter2) [reply .github/workflows/unfolding.yml:20]: Yes there is the possibility to write custom "actions" and I wasted some hour with this already. I was not able to get the outputs from the custom actions so in the end I just copied the code. But my attempt is still there in an older commit, see:
https://github.com/davidwalter2/WRemnants/blob/2f1b6e5acc930f925824601d903bb737b5398c3c/.github/actions/setenv/action.yml

Comment (kdlong) [reply .github/workflows/unfolding.yml:20]: Ok fair, can waste more time in the future when we are motivated

**Pull request merged**